numpy/version.py

```diff
@@ -12,4 +12,30 @@
 release = 'dev0' not in version and '+' not in version
 short_version = vinfo['version'].split("+")[0]
 
-del get_versions, vinfo
+
+def make_version_tuple() -> tuple[int, int, int]:
+    """
+    We want to expose a numeric version tuple to make it easier for
+    dependencies to have conditional code without having to parse our version
+    string.
+
+    1) It can be tricky as 1) you don't want to compare string. 2) it can be
+    tempted to just split on dot and map convert to int, but that will fail on
```

Comment:
```suggestion
    It can be tricky as 1) you don't want to compare string. 2) it can be
    tempting to just split on dot and map convert to int, but that will fail on
```